{
  "uid" : "f0997539eb472b88",
  "name" : "test_user_login",
  "fullName" : "TestCase.Test_Login.Test_Login1#test_user_login",
  "historyId" : "23588660a40a297b0a7e78ec71f1c1a5",
  "time" : {
    "start" : 1608737099000,
    "stop" : 1608737099000,
    "duration" : 0
  },
  "description" : "用例描述",
  "descriptionHtml" : "<p>用例描述</p>\n",
  "status" : "broken",
  "statusMessage" : "AttributeError: 'str' object has no attribute 'base_url11'",
  "statusTrace" : "self = <class 'Test_Login.Test_Login1'>\n\n    def setup_class(self):\n    \n        self.aa('aa')\n>       self.goto_url('',self.driver,'')\n\nPublic\\SetUp_TearDown.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = ''\nselenium_driver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"95fb711c-a158-40fd-9c50-b464d1941de5\")>\nbase_url = ''\n\n    def goto_url(self, selenium_driver, base_url=login_url): # 在初始化方法__init__()中定义驱动（driver），基本的URL(base_url)\n>       self.base_url11 = base_url                             # 和超时时间（timeout）\nE       AttributeError: 'str' object has no attribute 'base_url11'\n\nPublic\\Base.py:11: AttributeError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "_Class__pytest_setup_class",
    "time" : {
      "start" : 1608737099001,
      "stop" : 1608737099001,
      "duration" : 0
    },
    "status" : "broken",
    "statusMessage" : "AttributeError: 'str' object has no attribute 'base_url11'\n",
    "statusTrace" : "  File \"E:\\Python\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"E:\\Python\\lib\\site-packages\\_pytest\\fixtures.py\", line 1124, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"E:\\Python\\lib\\site-packages\\_pytest\\fixtures.py\", line 921, in call_fixture_func\n    fixture_result = next(generator)\n  File \"E:\\Python\\lib\\site-packages\\_pytest\\python.py\", line 799, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"E:\\Python\\lib\\site-packages\\_pytest\\python.py\", line 732, in _call_with_optional_argument\n    func(arg)\n  File \"E:\\GitTest\\Pytest126mail\\Public\\SetUp_TearDown.py\", line 29, in setup_class\n    self.goto_url('',self.driver,'')\n  File \"E:\\GitTest\\Pytest126mail\\Public\\Base.py\", line 11, in goto_url\n    self.base_url11 = base_url                             # 和超时时间（timeout）\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "shouldDisplayMessage" : true,
    "attachmentsCount" : 0,
    "hasContent" : true,
    "stepsCount" : 0
  } ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "登录套件"
  }, {
    "name" : "feature",
    "value" : "登录模块"
  }, {
    "name" : "parentSuite",
    "value" : "TestCase"
  }, {
    "name" : "subSuite",
    "value" : "Test_Login1"
  }, {
    "name" : "host",
    "value" : "DESKTOP-VTQ1542"
  }, {
    "name" : "thread",
    "value" : "1136-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "TestCase.Test_Login"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "f0997539eb472b88.json",
  "parameterValues" : [ ]
}